octodns/processor/templating.py

```diff
@@ -62,9 +62,9 @@ def __init__(self, id, *args, context={}, **kwargs):
     def process_source_zone(self, desired, sources):
         sources = sources or []
         zone_params = {
-            'zone_name': desired.decoded_name,
-            'zone_decoded_name': desired.decoded_name,
-            'zone_encoded_name': desired.name,
+            'zone_name': desired.decoded_name.rstrip('.'),
+            'zone_decoded_name': desired.decoded_name.rstrip('.'),
+            'zone_encoded_name': desired.name.rstrip('.'),
```

Comment:
Generally octoDNS removes the trailing `.` w/`[:-1]`, probably best to do that, based on a param, for consistency's sake.

Reply:
Fixed.